db/migrate/20150420223003_create_admins.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAdmins < ActiveRecord::Migration
+  def change
+    create_table :admins do |t|
+      t.string :username
+      t.string :password
```

Comment:
There is another task on the board to add authorization with bcrypt. We can add a migration to remove the password column from the database then.